### DISCUSSION TITLE
Aerosol model

### DISCUSSION THREAD

**Discussion by ahmedahmed2056** [firemodels/fds#13535] [General]
Hello

I have a model of a small trees fire and in front of it is a building. The model is 20 m x 25 m x 10 m. and tThe dimensions of the studied building is 7 m x 3.8 m x 3.5m. whenever i make the soot as Aerosol it gives ( numerical unstability ) error.
All the mesh boundaries are open and it didn't work so i changed to make a periodic in x direction and open in the rest and still the same error. Any other possibilities or possible solutions ?

![image](https://github.com/user-attachments/assets/6e9d40fc-3279-41bc-bf4e-6b3cc4ac41fe)

**Comment by rmcdermo** [comment]
Input file?

**Comment by lihy213** [comment]
Please upload your input file and tell if it was solved?
